Apps/JobScheduler/src/Properties/launchSettings.json

```diff
@@ -11,7 +11,7 @@
     "Healthgateway.JobScheduler": {
       "commandName": "Project",
       "launchBrowser": true,
-      "launchUrl": "hangfire",
```

Comment:
Why was this changed?

Reply:
hangfire is now the root route for the site.